gradle.properties

```diff
@@ -22,13 +22,11 @@
 # The releaseQualifier uses the following conventions:
```

Comment:
We don't use these other releaseQualifiers so can we remove them?

Reply:
Can, but I'm not ready to as part of this change.